helpers/database-import-scripts/rnacentral/generate_rnacentral_json.py

```diff
@@ -259,38 +325,48 @@ def get_publications(genome_sample_accession, reported_project, insdc_accession)
     else:
         # keep going through levels of samples until we get to the raw read samples
         while samples_to_check:
+            infinite_loop_protection += 1
+            if infinite_loop_protection > 20:
```

Comment:
Maybe 20 is too much? If loads of entries get to more than 10 it will slow down the process a lot, right?

Reply:
This is only meant to break an (unlikely) infinite loop. I can reduce it, chose 20 just to be safe. A normal record should never get anywhere close to 10 and if for some reason it does, that's fine.